ios/Helio.xcodeproj/project.pbxproj

```diff
@@ -2154,7 +2154,7 @@
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CLANG_ENABLE_MODULES = YES;
-				CODE_SIGN_ENTITLEMENTS = BetterSocial/BetterSocial.entitlements;
+				CODE_SIGN_ENTITLEMENTS = BetterSocialDevRelease.entitlements;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, the change in the `CODE_SIGN_ENTITLEMENTS` path from `BetterSocial/BetterSocial.entitlements` to `BetterSocialDevRelease.entitlements` is likely for different build configurations. Make sure the new entitlement file exists and has the correct settings for the dev release configuration.
```diff
- CODE_SIGN_ENTITLEMENTS = BetterSocial/BetterSocial.entitlements;
+ CODE_SIGN_ENTITLEMENTS = BetterSocialDevRelease.entitlements;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->